igvm/vm.py

```diff
@@ -151,6 +151,23 @@ def set_memory(self, memory):
         """Resizes the host memory."""
         self.hypervisor.vm_set_memory(self, memory)
 
+    def set_hostname(self, new_hostname, transaction=None):
+        """Changes the hostname of a VM"""
+        self.previous_hostname = self.dataset_obj['hostname']
+
+        self.dataset_obj['hostname'] = new_hostname
+        self.check_serveradmin_config()
+
+        self.dataset_obj.commit()
+
+        if transaction:
+            transaction.on_rollback('reset_hostname', self.revert_hostname)
```

Comment:
lets be consistent and dont mix names. decide for either `reset_hostname` or `revert_hostname`

Reply:
Oops, this was some leftover.